ur5_moveit_config/launch/moveit_rviz.launch

```diff
@@ -11,6 +12,7 @@
   <node name="$(anon rviz)" launch-prefix="$(arg launch_prefix)" pkg="rviz" type="rviz" respawn="false"
 	args="$(arg command_args)" output="screen">
     <rosparam command="load" file="$(find ur5_moveit_config)/config/kinematics.yaml"/>
+    <param name="arm_prefix" value="$(arg arm_prefix)"/>
```

Comment:
> The only way to fix it is to add that parameter at that specific location. I'm not sure exactly how moveit_plugins are loaded or where they're loaded into.. I've taken a look but haven't seen anything showing where moveit plugins are loaded in yet.

The plugins can be loaded twice (in a normal MoveIt config pkg): once by `move_group`, and once by (I guess) the RViz MoveIt plugin. As `URKinematicsPlugin::initialize(..)` instantiates a private `NodeHandle`, this makes the instance initialised by RViz look for parameters under the `/rviz_whatever_../` ns. The other instance looks for the parameter under `/move_group/`.

I'm not entirely sure how things are supposed to work actually, especially with `kinematics.yaml` being loaded under the `$(arg robot_description)_kinematics` namespace (by `planning_context.launch`), which is completely ignored by IK plugins, afaict.

However, I think adding the `arm_prefix` parameter to `kinematics.yaml` (at root level) would be much cleaner (also: how is the plugin otherwise supposed to retrieve this parameter if only `move_group.launch` is started (as is the case on *headless* robots)).